### ISSUE TITLE
It seems not working stably with Openwrt

### ISSUE THREAD

**Issue by Doublefire-Chen** [Aircoookie/WLED#2784]
### What happened?

I am using my Openwrt soft router as a DHCP server, but sometimes I can't open the management web with the ip address. Ping is success, nmap can't scan any port open on my ESP32. But when I am using a normal router with DHCP model, everything is working well.

### To Reproduce Bug

Let EDP32 connect to a router which is operating on  Openwrt system, then can't open the management. This problem is not all the time, sometimes you can open the page. So please let me describe it as not stable.

### Expected Behavior

Management page is not working.

### Install Method

Binary from WLED.me

### What version of WLED?

WLED 0.13.3(Built number I don't know as I can't open the page right now)

### Which microcontroller/board are you seeing the problem on?

ESP32

### Anything else?

Board: ESP32 Devkit v1
![Screen Shot 2022-09-16 at 14 46 30](https://user-images.githubusercontent.com/72138342/190574584-a9a091cb-3d00-41ea-9c85-637b6cab8c9b.png)
![Screen Shot 2022-09-16 at 14 48 47](https://user-images.githubusercontent.com/72138342/190574593-1146e7ae-f2a9-4cd1-8e24-0bcf36d0532e.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Doublefire-Chen** [comment]
Just moment I installed the system with bin file, same problem.

**Comment by Doublefire-Chen** [comment]
Just moment I installed the WLED 0.13.1 system with bin file, same problem.

**Comment by Doublefire-Chen** [comment]
I remember last time I solve this problem by restarting the router.

**Comment by Doublefire-Chen** [comment]
> I remember last time I solve this problem by restarting the router.

Just moment I solved this by restarting the router, but I am afraid when this problem happen again.

**Comment by Aircoookie** [comment]
Thank you for the issue report. Just to confirm that the connection issue is indeed caused by DHCP, have you tried if the issue occurs after setting a static IP within WLED settings ?

**Comment by Doublefire-Chen** [comment]
> Thank you for the issue report. Just to confirm that the connection issue is indeed caused by DHCP, have you tried if the issue occurs after setting a static IP within WLED settings ?

This issue occurs randomly, and now I am using it well. I remember once I tried with static ip connecting my soft router when this issue happened, but did't work.

**Comment by Doublefire-Chen** [comment]
It is a quite annoying thing.
Now it happened again and I noticed that the blue led on my ESP32 is twinkling all the time right now.

**Comment by Doublefire-Chen** [comment]
> It is a quite annoying thing. Now it happened again and I noticed that the blue led on my ESP32 is twinkling all the time right now.

I just solve it by restarting my router and ESP32, and I find that it is not matter with the blue led the board as it still twinkle though it is working well. I don't know when is next time.😭 Maybe I should install a new version of Openwrt to prove it is the problem of soft router's system.

**Comment by blazoncek** [comment]
Just get a good router from Ubiqiti, MikroTik or the like.
You can get EdgeRouter X for less than $50.

**Comment by mahipat99** [comment]
I have the same issue. ESP8266 goes offline randomly
Tried resetting/upgrading router FW multiple times, disabled blynk, hue, etc. , changed power supply / ESP 
nothing helps

As soon as I restart wifi radio, the device comes online but then goes offline randomly.

The only workaround I found is static IP from WLED itself, I have tried setting static IP from the router, too, but it also didn't work
one more thing, I have two routers, one is router+AP, and another is dumb AP(disabled DHCP). Workaround works only on router+AP; same disconnect issue with dumb AP.

FW - 22.03.0 (Mi Router 4A)
